index.html

```diff
@@ -33,12 +33,12 @@ <h1 class="logo-heading">Fun Bus</h1>
   <div class="container home">
     <header class="intro">
       <img src="img/fun-bus.jpg" alt="bus in the sand">
-      <h2>Welcome To Fun Bus!</h2>
+      <h2 class='SPIN' >Welcome To Fun Bus!</h2>
       <p>Traveling expedition modern, clean webdesign blogger clean website theme website modern. Design pretty design, travelblogger adventure WordPress wanderlust theme blogger website expedition theme travelblogger. Adventure fun traveler pretty design website expedition.</p>
     </header>
     <section class="content-section">
       <div class="text-content">
-        <h2>Let's Go!</h2>
+        <h2 class='ER'>Let's Go!</h2>
```

Comment:
you can do this without giving these a class, just by referencing the element itself (h2 in this case)